### PR TITLE
[FIX] web: qunit detects remaining body elements

### PR DESCRIPTION
**Pull request by Arcasias**

### CODE REVIEW

File: addons/web/static/tests/helpers/qunit_config.js [hunk 1]
```diff
@@ -45,6 +45,24 @@ var sortButtonAppended = false;
  * browser_js is closed as soon as an error is logged.
  */
 const errorMessages = [];
+/**
+ * List of elements tolerated in the body after a test. The property "keep"
+ * prevents the element from being removed (typically: qunit sutie elements).
```
Comment (aab-odoo) [inline addons/web/static/tests/helpers/qunit_config.js:50]: sutie -> suite

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by Xavier-Do** [comment]
robodoo retry

**Pull request closed without merging**